.github/workflows/publish-draft-releases.yml

```diff
@@ -229,7 +229,7 @@ jobs:
         if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
         name: fetch and chmod polkadot
         run: |
-          curl -L -o $HOME/.local/bin/polkadot https://github.com/paritytech/polkadot/releases/download/v0.9.16/polkadot
+          curl -L -o $HOME/.local/bin/polkadot https://github.com/paritytech/polkadot/releases/download/v0.9.17-rc4/polkadot
```

Comment:
I don't think we should use rc releases in a PR merged into production.

Reply:
It's parity's problem, I don't know why they issued a release on a rc branch.
Originally, I used `v0.9.16`, but our CI could not find the binary.